app/scripts/controllers/globalPromotions.ts

```diff
@@ -138,6 +144,53 @@ class GlobalPromotionsCtrl {
   isActive(promotion: GlobalPromotion): boolean {
     return this.globalPromotionService.isPromotionActive(promotion);
   }
+
+  setOrder(field: string) {
+    if (this.orderByField === field) {
+      this.reverseSort = !this.reverseSort;
+    } else {
+      this.orderByField = field;
+      this.reverseSort = false;
+    }
+  }
+
+  getStatusForSorting(promotion: GlobalPromotion): string {
+    return this.isActive(promotion) ? 'Active' : 'Inactive';
+  }
+
+  get sortField(): string | ((promotion: GlobalPromotion) => string) {
+    return this.orderByField === 'status'
+      ? this.getStatusForSorting
```

Comment:
Instead of having to bind the method, I would do this:

Or even inline `getStatusForSorting` like this: `? (promotion: GlobalPromotion) => this.isActive(promotion) ? 'Active' : 'Inactive'`

```suggestion
      ? (promotion: GlobalPromotions) => this.getStatusForSorting(promotion)
```